### PR TITLE
Fix Server Installer

### PR DESCRIPTION
**Pull request by calmilamsy**
Server installer is fixed and the server works so far.

Currently downloads the server jar from a site I have ran for a while.

### CODE REVIEW

File: src/main/java/net/fabricmc/installer/server/ServerInstaller.java [hunk 1]
```diff
@@ -18,16 +18,7 @@
...
-import java.io.BufferedWriter;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.OutputStream;
-import java.io.OutputStreamWriter;
+import java.io.*;
```
Comment (valoeghese) [inline src/main/java/net/fabricmc/installer/server/ServerInstaller.java]: this all import is my only nitpick

File: src/main/java/net/fabricmc/installer/server/ServerPostInstallDialog.java [hunk 2]
```diff
@@ -22,16 +22,10 @@
...
 import java.awt.Toolkit;
-import java.io.BufferedInputStream;
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import java.io.*;
```
Comment (valoeghese) [inline src/main/java/net/fabricmc/installer/server/ServerPostInstallDialog.java]: it appears again

**Review by valoeghese — CHANGES_REQUESTED** [review]

**Comment by valoeghese** [comment]
inb4 there were already star imports somewhere in the code

File: src/main/java/net/fabricmc/installer/server/ServerInstaller.java (hunk 1 above)
Comment (calmilamsy) [reply src/main/java/net/fabricmc/installer/server/ServerInstaller.java]: I went through *all* files and removed star and unused imports.

File: src/main/java/net/fabricmc/installer/Handler.java [hunk 3]
```diff
@@ -16,11 +16,10 @@
 
 package net.fabricmc.installer;
 
-import java.awt.Color;
-import java.awt.Container;
-import java.awt.FlowLayout;
-import java.io.IOException;
-import java.util.function.Consumer;
+import net.fabricmc.installer.util.ArgumentParser;
```
Comment (Chocohead) [inline src/main/java/net/fabricmc/installer/Handler.java:19]: I don't see why these imports need to be moving at all

Comment (calmilamsy) [reply src/main/java/net/fabricmc/installer/Handler.java:19]: Woops, blame Intellij for that.

**Review by valoeghese — APPROVED** [review]

**Pull request merged**